pkg/aws/client.go

```diff
@@ -0,0 +1,48 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+)
+
+// Client provides the single API client to make operations call to aws services
+type Client struct {
+	ec2 *ec2.Client
```

Comment:
Can we define an interface with the calls we consume and test the `Client`?

Reply:
yes i did in an upcoming pr which breaks things down w unit tests